runtime/ftplugin/help.lua

```diff
@@ -82,5 +91,6 @@ vim.keymap.set('n', 'g==', function()
 end, { buffer = true })
 
 vim.b.undo_ftplugin = (vim.b.undo_ftplugin or '')
-  .. '\n exe "nunmap <buffer> gO" | exe "nunmap <buffer> g=="'
+  .. '\n exe "nunmap <buffer> gO" | exe "nunmap <buffer> g==" | '
+  .. '\n exe "nunmap <buffer> ]]" | exe "nunmap <buffer> [["'
```

Comment:
Why are the second and third commands separated by both a `|` and a `\n`?

Reply:
eh, better safe than sorry